addons/web/static/src/webclient/company_service.js

```diff
@@ -115,14 +119,37 @@ export const companyService = {
                     }
                 }
 
+                let stayOnCurrentPage = true;
+
+                if (currentModel && currentId && 'company_id' in await env.services.field.loadFields(currentModel)) {
+                    try {
+                        await env.services.orm.call(
```

Comment:
I'm not sure if it is good practice to use `env.services`?

Reply:
umm what would be the alternative ?

Reply:
You need to add the services that you want to use (in another service) on the dependency list (check line 55).
And you add on the start (line 56) the service that you added, and you can use it directly (check how router, action or user ar used).

Reply:
This should be done to be sure that when starting the company service, the orm service is already started.

Reply:
done